unpackaged/config/rd2_post_config/layouts/npe03__Recurring_Donation__c-Enhanced Recurring Donations Layout.layout

```diff
@@ -133,6 +133,9 @@
         <quickActionListItems>
             <quickActionName>NewTask</quickActionName>
```

Comment:
Since we're adding a namespace to the one below, this one should also get a namespace. I noticed that it causes issues for some customer orgs when it's not there.
```suggestion
            <quickActionName>%%%NAMESPACE%%%NewTask</quickActionName>
```